_config.yml

```diff
@@ -10,6 +10,8 @@ defaults:
 
 baseurl: /rsd-engineeringcourse
 
+remote_theme: dpshelio/indigo-jekyll
```

Comment:
is it worth having this in UCL-ARC org?

Reply:
I think that ideally that lives under ucl (even if ARC maintains it). I'd probably move it there once it's done and update accordingly.